instrumentation/opentelemetry-instrumentation-cherrypy/src/opentelemetry/instrumentation/cherrypy/package.py

```diff
@@ -0,0 +1,3 @@
+_instruments = ("cherrypy >= 1.0",)
```

Comment:
Please also pin the upper version, there can be breaking changes b/w major version changes.